docs/UserGuide.adoc

```diff
@@ -49,6 +49,11 @@ Some example commands you can try:
 
 Refer to <<Features>> for details of each command.
 
+To begin using administrator commands, login to the default master account with the following command: +
+`login user/master pass/Pass#123 userdata/masterconfig` +
```

Comment:
It would be good to change it to `userdata/masterconfig.xml`.
This is to stay consistent with the save feature which specifies the file extension.
I have already updated the file name to `masterconfig.xml` for the actual file.

Reply:
Alright, added. Thanks for the update too.